R/calculate_taxes.R

```diff
@@ -241,11 +251,40 @@ taxsim_calculate_taxes <- function(.data, marginal_tax_rates = 'Wages', return_a
                         header = T,
                         sep = ","))
 
+    message("Connected to TAXSIM server and downloaded tax data.")
+
+  } else if (interface == "wasm") {
+
+    # connect to js and wasm files
+    wasm_path   <- system.file("webassembly/taxsim.wasm", package = "usincometaxes")
+    js_path     <- system.file("javascript/taxsim.js",    package = "usincometaxes")
```

Comment:
I would suggest putting the js/wasm file in the same directory, possibly named `ext` or `vendor`. The reason being that the js file tries to load the wasm file from the same path, and it would make it easier to test the wasm build using node/d8/etc if they were in the same folder.

In the future the R V8 lib may implement fetching, in which case the extra steps currently used for `assign("wasmBinary")` would no longer be required.

Reply:
Fixed with thomascwells/usincometaxes@4a22d66575878e191ff6cc625e13abf6b45baee6